docs/03-endpoints/api-admin/permissions.md

```diff
@@ -222,8 +223,53 @@ Therefore, it is not possible to create new default object access permissions
 for the ProjectAdmin and ProjectMember groups of a project. 
 However, the default permissions set for these groups can be modified; see below for more information.
 
+### Updating an existing Default Object Access Permission
+
+- `PUT: /admin/permissions/doap/<doap_permissionIri>` to change the attributes of an existing default object
+  access permission, identified by its IRI `<doap_permissionIri>`.
+
+This is an example of a request body to update an existing default object access permission:
+
+```json
+{
+  "forProperty" : "http://api.dasch.swiss/ontology/00FF/images/v2#hasTitle",
+  "forResourceClass": "http://api.dasch.swiss/ontology/0803/incunabula/v2#Book",
+  "forGroup": null, 
+  "hasPermissions": [
+    {
+      "additionalInformation": "http://www.knora.org/ontology/knora-admin#ProjectMember",
+      "name": "D",
+      "permissionCode": 7
+    }
+  ]
+}
+```
+
+All attributes of the default object access permission are optional and may be combined.
+
+!!! note
```

Comment:
To put more emphasis on it, I would perhaps choose a warning (https://squidfunk.github.io/mkdocs-material/reference/admonitions/#+type:warning)

```suggestion
!!! warning
```